vendor.conf

```diff
@@ -39,5 +39,5 @@ github.com/pkg/profile 5b67d428864e92711fcbd2f8629456121a56d91f
 github.com/tonistiigi/fsutil 195d62bee906e45aa700b8ebeb3417f7b126bb23
 github.com/stevvooe/continuity 86cec1535a968310e7532819f699ff2830ed7463
 github.com/dmcgowan/go-tar 2e2c51242e8993c50445dab7c03c8e7febddd0cf
-github.com/hashicorp/go-immutable-radix 8e8ed81f8f0bf1bdd829593fdd5c29922c1ea990
-github.com/hashicorp/golang-lru a0d98a5f288019575c6d1f4bb1573fef2d1fcdc4
\ No newline at end of file
+github.com/hashicorp/go-immutable-radix 826af9ccf0feeee615d546d69b11f8e98da8c8f1 git://github.com/tonistiigi/go-immutable-radix.git
```

Comment:
Do you plan to propose https://github.com/tonistiigi/go-immutable-radix/commit/826af9ccf0feeee615d546d69b11f8e98da8c8f1 to the upstream?